lib/foods/foods.js

```diff
@@ -0,0 +1,64 @@
+$(document).ready(function() {
+  $(loadFoods)
+})
+
+function loadFoods() {
+  $.get('https://obscure-harbor-85447.herokuapp.com/api/v1/foods')
+  .then(function(foods) {
+    foods.forEach(function(food) {
+      $('.foods-table').append(
+        `<tr class=row${food.id}><td>`+ food.name + '</td>' +
+        '<td>' + food.calories + '</td>' +
+        `<td><input id=${food.id} type='button' class='delete-row' value='Delete'/></td>` +
+        "</tr>"
+      )
+    })
+  })
+}
+
+function deleteFood(e) {
+  $.ajax({
+    type: 'DELETE',
+    url:'https://obscure-harbor-85447.herokuapp.com/api/v1/foods/' + e.target.id,
+    success: $(`tr.row${e.target.id}`).remove()
+  })
+}
+
+$('form').submit(function(e) {
```

Comment:
what if there's more than one form on the page? selecting your form in this way may cause problems in the future. can we be more selective here?

Reply:
Okay so scope it down to .foods-table .row# should work